src/platform/site-wide/sass/formation-overrides/_utilities.scss

```diff
@@ -0,0 +1,46 @@
+@import "functions";
+
+// Content size helpers
+@mixin allow-layout-classes {
+  @include margin(null auto);
+  &.align-left {
+    @include media($medium-screen) {
+      margin-right: rem-override(2em) !important;
+      margin-top: rem-override(0.5em) !important;
```

Comment:
I'm just grabbing styles that have a `rem` value and applying the overrides function. All the other styles can be left out. It will load with the original stylesheet.